website/src/components/hero/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { ReactNode } from 'react';
+import { DecorationIsolation, cn, CodegenIcon } from '@theguild/components';
+import Image from 'next/image';
+
+import codegenHeroBadge from './codegen-badge.svg';
+
+export function Hero(props: { children: ReactNode; className?: string }) {
+  return (
+    <div
+      className={cn(
+        'relative isolate flex max-w-[90rem] flex-col items-center justify-center gap-6 overflow-hidden rounded-3xl bg-blue-400 px-4 py-6 sm:py-12 md:gap-8 lg:py-24',
+        props.className
+      )}
+    >
+      <DecorationIsolation className="-z-10">
+        <CodegenIcon className="absolute left-[-180px] top-[calc(50%-180px)] size-[360px] fill-[url(#codegen-hero-gradient)] stroke-white/10 stroke-[0.1px] md:hidden lg:left-[-649px] lg:top-[-58px] lg:size-[1047px] xl:block" />
+        <CodegenIcon className="absolute right-[-350px] top-2 hidden size-[672px] fill-[url(#codegen-hero-gradient)] stroke-white/10 stroke-[0.1px] md:block" />
+        <svg>
+          <defs>
+            <linearGradient id="codegen-hero-gradient" x1="0%" y1="0%" x2="100%" y2="100%">
+              <stop offset="11.66%" stopColor="rgba(255, 255, 255, 0.10)" />
+              <stop offset="74.87%" stopColor="rgba(255, 255, 255, 0.30)" />
+            </linearGradient>
+          </defs>
+        </svg>
+      </DecorationIsolation>
+      <Image priority src={codegenHeroBadge.src} alt="" width="96" height="96" />
+      {props.children}
+    </div>
+  );
+}
+
+export function HeroLinks(props: { children: ReactNode }) {
+  return (
+    <div className="relative z-10 flex flex-col justify-center gap-2 px-0.5 sm:flex-row sm:gap-4">{props.children}</div>
+  );
+}
+
+export function HeroFeatures(props: { children: ReactNode }) {
+  return (
+    <ul className="mx-auto flex list-none flex-col gap-x-6 gap-y-2 text-sm font-medium md:flex-row [&>li]:flex [&>li]:items-center [&>li]:gap-2">
```

Comment:
```suggestion
    <ul className="mx-auto flex list-none max-md:flex-col gap-x-6 gap-y-2 text-sm font-medium [&>li]:flex [&>li]:items-center [&>li]:gap-2">
```

Reply:
Would you like to add a lint rule to optimize className props for length?

https://tailwindcss.com/docs/responsive-design#working-mobile-first

I find it a bit easier to work mobile-first and use the short prefixes instead of `max-X`

Reply:
in tailwindcss plugin? they cannot even merge your PR 😅